<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="userAVGFeedbackButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>User Average Feedback</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userBMIButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>User BMI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userBloodPressureButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>User Blood Pressure</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current diet plan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="currDietlistWidget"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="personalInfoGroupBox">
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>700</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string>Personal Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="usernameLabel">
             <property name="styleSheet">
              <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;</string>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="inputUserName">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="passwordLabel">
             <property name="styleSheet">
              <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;</string>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="inputPasslabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="fullNameLabel">
             <property name="styleSheet">
              <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Full Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="inputFullNamelabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="ageLabel">
             <property name="styleSheet">
              <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="inputAgelabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="genderLabel">
             <property name="styleSheet">
              <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Gender</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="inputGenderlabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Users</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="usersAverageBMIButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Users average BMI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="usersAvergeFeedBackButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Users Average Feedback</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
